compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -585,6 +585,7 @@ generateStablePackages lfVersion fp = do
         let dalfs = map (fp </>) $ concat
                 [ map ("daml-prim" </>)
                     [ "DA-Internal-Erased.dalf"
+                    , "DA-Internal-Error.dalf"
```

Comment:
maybe worth considering moving this from `Error` to `Exception`. It seems a bit odd to have builtin exceptions defined in an `Error` module. But we can also postpone that until we finalize builtin exceptions.

Reply:
We already have a `DA.Internal.Exception`. Maybe some other name? :-)

Reply:
`DA.Exception.Types` similar to the other `.Types` modules?

Reply:
`DA.Internal.Exception.Types` then :-)

Reply:
Is it internal though? Those types seem like they should be public API and they are by being serializable.